client.go

```diff
@@ -457,3 +464,29 @@ func checkContainerRunning(id string) (types.Container, error) {
 
 	return container, nil
 }
+
+func checkContainerExists(containerID string) error {
```

Comment:
The name of this function suggests a boolean return value, to be honest. I'd simply return true or false, since the given error right now is not particularly useful.

Reply:
I agree with you. I will change the output to boolean.